api/handlers.go

```diff
@@ -194,10 +194,102 @@ const FEED_SUGGESTER_JSON_V3 = `
         }
 `
 
+const FEED_SUGGESTER_JSON_V4_WITH_SINGLE_BLOG_POST = `
+{
+	"name": "SortSuggester",
+	"specs": [
+		{
+			"name": "SortSuggester",
+			"specs": [
+				{
+					"name": "RoundRobinSuggester",
+					"specs": [
+						{
+							"name": "CompletionSuggester",
+							"specs": [
+								{
+									"name": "LimitSuggester",
+									"specs": [
+										{
+											"name": "DataCollectionsSuggester",
+											"filters": [
+												{
+													"filter_selector": 1,
+													"args": [
+														"DAILY_LESSON"
+													]
+												}
+											]
+										}
+									],
+									"args": [
+										"1"
+									]
+								},
+								{
+									"name": "SortSuggester",
+									"specs": [
+										{
+											"name": "DataCollectionsSuggester",
+											"filters": [
+												{
+													"filter_selector": 1,
+													"args": [
+														"DAILY_LESSON"
+													]
+												}
+											]
+										},
+										{
+											"name": "DataContentUnitsSuggester",
+											"filters": [
+												{
+													"filter_selector": 0,
+													"args": [
+														"VIDEO_PROGRAM_CHAPTER",
+														"MEAL",
+														"FRIENDS_GATHERING",
+														"ARTICLE"
+													]
+												}
+											]
+										},
+										{
+											"name": "LimitSuggester",
+											"specs": [
+												{
+													"name": "DataContentUnitsSuggester",
+													"filters": [
+														{
+															"filter_selector": 0,
+															"args": [
+																"BLOG_POST"
+															]
+														}
+													]
+												}
+											],
+											"args": [
+												"1"
+											]
+										}
+									]
+								}
+							]
+						}
+					]
+				}
+			]
+		}
+	]
+}
```

Comment:
I think the following is equivalent:




{
  "name": "SortSuggester",
  "specs": [
    {
      "name": "CompletionSuggester",
      "specs": [
        {
          "name": "LimitSuggester",
          "specs": [
            {
              "name": "DataCollectionsSuggester",
              "filters": [
                {
                  "filter_selector": 1,
                  "args": [
                    "DAILY_LESSON"
                  ]
                }
              ]
            }
          ],
          "args": [
            "1"
          ]
        },
        {
          "name": "SortSuggester",
          "specs": [
            {
              "name": "DataCollectionsSuggester",
              "filters": [
                {
                  "filter_selector": 1,
                  "args": [
                    "DAILY_LESSON"
                  ]
                }
              ]
            },
            {
              "name": "DataContentUnitsSuggester",
              "filters": [
                {
                  "filter_selector": 0,
                  "args": [
                    "VIDEO_PROGRAM_CHAPTER",
                    "MEAL",
                    "FRIENDS_GATHERING",
                    "ARTICLE"
                  ]
                }
              ]
            },
            {
              "name": "LimitSuggester",
              "specs": [
                {
                  "name": "DataContentUnitsSuggester",
                  "filters": [
                    {
                      "filter_selector": 0,
                      "args": [
                        "BLOG_POST"
                      ]
                    }
                  ]
                }
              ],
              "args": [
                "1"
              ]
            }
          ]
        }
      ]
    }
  ]
}

Reply:
fixed